djangogirls/mysite/settings.py

```diff
@@ -26,9 +27,9 @@
 )
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
 
-ALLOWED_HOSTS = ['127.0.0.1', '.pythonanywhere.com']
+ALLOWED_HOSTS = ['127.0.0.1', '.pythonanywhere.com', '.heroku.app']
```

Comment:
Why `.app`? I haven't heard of them using this domain. Can you show me a document that describes such a possibility?

Reply:
It's my mistake